interfaces/Portalicious/src/app/services/export.service.ts

```diff
@@ -165,24 +165,6 @@ export class ExportService {
     };
   }
 
-  async getDuplicateCheckAttributes(
-    projectId: Signal<number | string>,
-  ): Promise<string[]> {
-    // TODO: AB#30519 This is a temporary solution until we have a better way to get all project attribute with the `duplicateCheck` flag included.
```

Comment:
Did you also remove/update this item from the backlog?

Reply:
Good one! That had a look and that task is already done. So I think nothing needs to happen